pandas/tests/indexing/interval/test_interval_new.py

```diff
@@ -163,7 +163,7 @@ def test_loc_with_overlap(self, indexer_sl):
         tm.assert_series_equal(expected, result)
 
         msg = "'can only get slices from an IntervalIndex if bounds are"
-        " non-overlapping and all monotonic increasing or decreasing'"
+        # non-overlapping and all monotonic increasing or decreasing'
```

Comment:
likewise here

Reply:
Sure thing! I made the change to make it a string again, and retested the code
<img width="886" alt="image" src="https://user-images.githubusercontent.com/18587013/204201677-74016f4a-99d1-4bd8-9504-ac24956eb1ae.png">

I have made the change as part of: efe54026d6608b13160315b5f6560f3174657302